.github/workflows/ci_workflow.yml

```diff
@@ -0,0 +1,397 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the OpenColorIO Project.
+
+name: ci
+
+on:
+  # Note: Jobs are skipped when ONLY Markdown (*.md) files are changed
+  push:
+    branches:
+      - master
+      - RB-*
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    branches:
+      - master
+      - RB-*
+    paths-ignore:
+      - '**.md'
+
+  schedule:
+    # Full nightly build
+    - cron: "0 0 * * *"
+
+jobs:
+  # Linux jobs run in Docker containers, so the latest OS version is OK. macOS 
+  # and Windows jobs need to be locked to specific virtual environment 
+  # versions to mitigate issues from OS updates, and will require maintenance 
+  # as OS versions are retired.
+
+  # ---------------------------------------------------------------------------
+  # Linux
+  # ---------------------------------------------------------------------------
+  # Note: 'runs-on' defines the GH-hosted VM. 'container' is the Docker 
+  #       container where the build occurs.
+  # Note: 'container' images are hosted on DockerHub
+
+  linux:
+    name: Linux CentOS 7 VFX CY2019 ${{ matrix.build-desc }}
```

Comment:
If we do not control the versions, we should remove them from the name.

Reply:
We do actually control the Linux version, but it's controlled through the aswf container we spin up rather than through the GH-provided VM. The comment on Linux version is specific to the VM.